src/ctim/schemas/sighting.cljc

```diff
@@ -1,15 +1,28 @@
 (ns ctim.schemas.sighting
-  (:require [ctim.schemas.common :as c]
-            [ctim.schemas.relationship :as rel]
-            [ctim.schemas.vocabularies :as v]
-            #?(:clj  [flanders.core :as f :refer [def-entity-type def-eq def-map-type]]
-               :cljs [flanders.core :as f :refer-macros [def-entity-type def-eq def-map-type]])))
+  #?@
+  (:clj
+   [(:require
+     [ctim.schemas.common :as c]
+     [ctim.schemas.relationship :as rel]
+     [ctim.schemas.vocabularies :as v]
```

Comment:
Why do you put these namespaces in the the reader conditionals part? It's common to clj and cljs

Reply:
cider and clj-format do that when you automatically reformat the ns,
I wonder if I prefer manually maintain the conditionals or let the tools do it.